src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -278,7 +280,7 @@ public ReferenceDataContext(Loan loan, DeclareItemLostRequest request, String st
 
     public ReferenceDataContext withLostItemPolicy(LostItemPolicy lostItemPolicy) {
       this.lostItemPolicy = lostItemPolicy;
-      this.loan = loan.withLostItemPolicy(lostItemPolicy);
+      this. loan = loan.withLostItemPolicy(lostItemPolicy);
```

Comment:
```suggestion
      this.loan = loan.withLostItemPolicy(lostItemPolicy);
```

Reply:
this change was added to my PR by merging master incorrectly 
so, please check this : [here](https://github.com/folio-org/mod-circulation/pull/1150)